requirements/prod-requirements.txt

```diff
@@ -116,6 +117,7 @@ pexpect==4.7.0            # via ipython
 phonenumberslite==8.10.10
 pickleshare==0.7.5        # via ipython
 pillow==6.0.0
+pip-tools==4.0.0
```

Comment:
I don't think these should be added to prod?

Reply:
well apparently i missed this when reviewing https://github.com/dimagi/commcare-hq/pull/24968. will submit a cleanup PR